File: 083_Remove_Duplicates_from_Sorted_List/083_1.py
class ListNode:
     def __init__(self, x):
         self.val = x
         self.next = None

class Solution:
    def deleteDuplicates(self, head):
        """
        :type head: ListNode
        :rtype: ListNode
        """
        res = head
        while res is not None and res.next is not None:
            if res.val == res.next.val:
                res.next = res.next.next
            else:
                res = res.next
        return head

File: 083_Remove_Duplicates_from_Sorted_List/083_1.java
public class ListNode {
     int val;
     ListNode next;
     ListNode(int x) { val = x; }
}

class Solution {
    public ListNode deleteDuplicates(ListNode head) {
        ListNode res = head;
        while (res != null && res.next != null)
        {
            if (res.val == res.next.val)
                res.next = res.next.next;
            else
                res = res.next;
        }   
        return head;
    }